Tests/FluentAssertions.Specs/Primitives/NullableBooleanAssertionSpecs.cs

```diff
@@ -200,6 +200,22 @@ public void When_asserting_boolean_null_value_to_be_equal_to_different_nullable_
                 .WithMessage("Expected False because we want to test the failure message, but found <null>.");
         }
 
+        [Fact]
+        public void When_asserting_boolean_null_value_not_to_be_equal_to_same_boolean_null_value_should_fail()
```

Comment:
🔧 You don't need to repeat that it's a `boolean null value`. That's clear from the context and only makes the names harder to understand. Same for the other tests.